src/style.css

```diff
@@ -0,0 +1,102 @@
+body {
+  margin: 0;
+  font-family: Arial, "Helvetica Neue", Helvetica, sans-serif;
+  color: #fff;
+  font-size: 100px;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+}
+
+.block {
+  height: 300px;
+  min-width: 300px;
+  line-height: 300px;
+  text-align: center;
+  flex-grow: 1;
```

Comment:
you add `display: flex` here and centre vertically and horizontally
also add `flex-basis` property here